### DISCUSSION TITLE
Migracion Bootstrap 5

### DISCUSSION THREAD

**Discussion by adolfojam98** [gcba/Obelisco#239] [Preguntas y consultas]
Soy desarrollador de los proyectos Gestión Colaborativa y Solicitudes BA. Quisiera saber si están planeando migrar a Bootstrap 5, ya que la ASI no homologa más la versión 4. Si es así, ¿cuándo estaría disponible la nueva versión? Además, ¿existe alguna manera de utilizar Bootstrap 5 con el proyecto Obelisco?

Muchas Gracias!

**Comment by bayramtturgutt** [comment]
Dado que la ASI ya no homologa Bootstrap 4, es prudente planificar una migración a Bootstrap 5.

Integración con el Proyecto Obelisco:
Si deseas utilizar Bootstrap 5 en el proyecto Obelisco, puedes hacerlo siguiendo estos pasos generales:
Actualizar las Dependencias: Reemplaza la versión de Bootstrap 4 con Bootstrap 5 en las dependencias del proyecto.
Revisar y Adaptar el Código: Bootstrap 5 introduce cambios en las clases y funcionalidades. Es posible que necesites ajustar el código existente para que sea compatible con la nueva versión.
Pruebas y Ajustes: Realiza pruebas exhaustivas para garantizar que los cambios no afecten la funcionalidad y el diseño del proyecto.

**Comment by lautarorodriguez96** [comment]
Hola @adolfojam98  ¿Cómo estás? Tal como mencionas, el equipo está al tanto de esta nueva actualización y ya estamos  trabajando en la migración a la versión de Bootstrap 5. Te mantendremos informado por este medio sobre cualquier novedad o podes escribirnos a  equipoobelisco@buenosaires.gob.ar

Saludos.
